test/thread/src/fiber_guard_page.d

```diff
@@ -18,21 +22,34 @@ void main()
 {
     auto test_fiber = new Fiber(&stackMethod, stackSize);
 
+    auto getPrivateFiberField(string id)()
```

Comment:
I am a bit puzzled, why not use `__traits(getMember, id)` directly instead of this function ?

Reply:
'cut thats what LDC did ;) Maybe because its private? Feel free to change it as Im about to go to bed.

Reply:
[`getMember` bypasses visibility checks](https://github.com/dlang/dmd/blob/c6ab9c400f54704a1287facc2de03d7a10c10e33/src/dmd/traits.d#L959-L978)

But yeah, I'll raise a PR in LDC.